src/Yoti/Util/Profile/AnchorListConverter.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Yoti\Util\Profile;
+
+use Traversable;
+
+class AnchorListConverter
+{
+    public static function convert(Traversable $anchorList)
+    {
+        $yotiAnchorsMap = [];
+        $anchorConverter = new AnchorConverter();
```

Comment:
If _AnchorConverter_ is threadsafe then we don't need to create a new one every time we enter this method.

Reply:
Unfortunately no - The ASN1 and X509 object could be overwritten, so I will leave it as it is.